aws/hll/aws-s3-transfer-manager/README.md

```diff
@@ -34,3 +36,22 @@ AWS_PROFILE=<profile-name> RUST_LOG=trace cargo run --example cp s3://<my-bucket
 
 NOTE: To run in release mode add `--release/-r` to the command, see `cargo run -h`.
 NOTE: `trace` may be too verbose, you can see just this library's logs with `RUST_LOG=aws_s3_transfer_manager=trace`
+
+#### Flamegraphs
+
+See [cargo-flamegraph](https://github.com/flamegraph-rs/flamegraph) for more prerequisites and installation information.
```

Comment:
It would be super cool to upload these to a new folder in the design docs with info on the current state of the project.